Backend/src/DbManager/AuthManager.ts

```diff
@@ -0,0 +1,28 @@
+import { MysqlSequelizeAdapter } from "../db/Mysql/MysqlDbAdapter";
```

Comment:
Can we have all file name format for DbManager like example **todo.db-manager.ts**